android/app/build.gradle

```diff
@@ -336,7 +333,7 @@ if (!sentrySlug.contains('SENTRY_ORGANIZATION')) {
       // Default is enabled.
       // Only available v3.1.0 and above.
       autoInstallation {
-        enabled = true
+        enabled = false
```

Comment:
This is no longer needed with the upgraded SDK, which now includes a built-in Sentry runtime. We would only need this flag if we needed to customize the install somehow.